### PR TITLE
Static Analysis warnings

### PR DESCRIPTION
**Pull request by LianaHus**
fix https://github.com/ethereum/remix-ide/issues/2808
fix https://github.com/ethereum/remix-ide/issues/2820

### CODE REVIEW

File: src/app/ui/landing-page/landing-page.js [hunk 1]
```diff
@@ -233,8 +233,8 @@ export class LandingPage extends ViewPlugin {
             <div class="${css.logoContainer}">${img}</div>
           </div>
           <div class="${css.jumboBtnContainer} px-5">
-            <button class="btn btn-primary btn-lg mx-3" href="#" onclick=${() => learnMore()} role="button">Learn more</button>
-            <button class="btn btn-secondary btn-lg" onclick=${() => switchToPreviousVersion()}>Use previous version</button>
+            <button class="btn btn-primary mx-3" href="#" onclick=${() => learnMore()} role="button">Learn more</button>
+            <button class="btn btn-secondary" onclick=${() => switchToPreviousVersion()}>Use previous version</button>
```
Comment (LianaHus) [inline src/app/ui/landing-page/landing-page.js:237]: this is fixing home tab buttons for not native themes

File: src/app/tabs/staticanalysis/staticAnalysisView.js [hunk 2]
```diff
@@ -105,6 +107,9 @@ staticAnalysisView.prototype.run = function () {
   if (!this.view) {
     return
   }
+  const highlightLocation = (location, fileName) => {
+    this.sourceHighlighter.currentSourceLocationFromfileName(location, fileName)
```
Comment (yann300) [inline src/app/tabs/staticanalysis/staticAnalysisView.js:111]: if you have access to the plugin API from here, it'd be better to use it:
`await remix.call('editor', 'highlight', pos, 'browser/3_Ballot.sol')`
and see https://github.com/ethereum/remix-ide/pull/2815/files#diff-bad7952ccc3083125a4c5549afda1978R133 for example how to set highlight, remove a specific highlight and remove all highlights

Comment (yann300) [reply src/app/tabs/staticanalysis/staticAnalysisView.js:111]: probably that we should then move the debugger to using this API also

File: src/app/tabs/staticanalysis/staticAnalysisView.js [hunk 3]
```diff
@@ -206,7 +261,7 @@ staticAnalysisView.prototype.renderModules = function () {
     return yo`
       <div class="${css.block}">
         <input type="radio" name="accordion" class="w-100 d-none card" id="heading${categoryId}" onclick=${(e) => this.handleCollapse(e)}"/>
-        <label for="heading${categoryId}" style="cursor: pointer;" class="pl-3 h6 card-header d-flex justify-content-between font-weight-bold border-left px-1 py-2 w-100">
+        <label for="heading${categoryId}" style="cursor: pointer;" class="pl-3 card-heaader h6 d-flex justify-content-between font-weight-bold border-left px-1 py-2 w-100">
```
Comment (yann300) [inline src/app/tabs/staticanalysis/staticAnalysisView.js:264]: typo here `heaader h6`

File: src/app/tabs/staticanalysis/staticAnalysisView.js (hunk 2 above)
Comment (LianaHus) [reply src/app/tabs/staticanalysis/staticAnalysisView.js:113]: #2834

File: src/app/tabs/staticanalysis/staticAnalysisView.js (hunk 3 above)
Comment (LianaHus) [reply src/app/tabs/staticanalysis/staticAnalysisView.js:264]: thanks

File: src/app/tabs/staticanalysis/staticAnalysisView.js [hunk 4]
```diff
@@ -206,7 +261,7 @@ staticAnalysisView.prototype.renderModules = function () {
     return yo`
       <div class="${css.block}">
         <input type="radio" name="accordion" class="w-100 d-none card" id="heading${categoryId}" onclick=${(e) => this.handleCollapse(e)}"/>
-        <label for="heading${categoryId}" style="cursor: pointer;" class="pl-3 h6 card-header d-flex justify-content-between font-weight-bold border-left px-1 py-2 w-100">
+        <label for="heading${categoryId}" style="cursor: pointer;" class="pl-3 card-header h6 d-flex justify-content-between font-weight-bold border-left px-1 py-2 w-100">
           <span>${category[0].categoryDisplayName}</span>
```
Comment (ryestew) [inline src/app/tabs/staticanalysis/staticAnalysisView.js:265]: Do these need to be wrapped in a span?

**Pull request merged**